.github/workflows/scripts/slack_notifications/gem_notifier.rb

```diff
@@ -0,0 +1,89 @@
+# encoding: utf-8
```

Comment:
You may want to merge dev into your branch. This line will now raise a rubocop failure. Sorry I didn't notice this earlier!

Reply:
Ah! Thank you. I'll merge dev in and then re-request approval :)